packages/next-swc/crates/core/Cargo.toml

```diff
@@ -25,6 +26,10 @@ swc_node_base = "0.5.1"
 swc_stylis = "0.43.0"
 tracing = {version = "0.1.28", features = ["release_max_level_off"]}
 regex = "1.5"
+relay-compiler = { git = "https://github.com/facebook/relay", tag="v13.0.1" }
```

Comment:
Can you make this dependency non-wasm only or move this to `napi`?

Reply:
As all file operations will fail on wasm build, realy will not be supported for wasm builds.